front/src/modules/ui/components/buttons/ButtonGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import { ButtonPosition, ButtonVariant } from './Button';
+
+const StyledButtonGroupContainer = styled.div`
+  border-radius: 8px;
+  display: flex;
+`;
+
+type ButtonGroupProps = {
+  children: React.ReactElement[];
+  variant?: ButtonVariant;
```

Comment:
@FelixMalfait I think `size` prop should be added too, you can create `ButtonGroupProps` directly from `ButtonProps` by picking the prop you want to keep with `Pick`